app/src/routing.js

```diff
@@ -0,0 +1,14 @@
+(function () {
+    angular
+        .module('webshopModule',['ngRoute'])
```

Comment:
Please only _reference_ the module defined in `webshopModule.js`. You can specify the module dependencies there.